mythril/laser/ethereum/svm.py

```diff
@@ -229,12 +236,15 @@ def exec(self, create=False, track_gas=False) -> Union[List[GlobalState], None]:
         """
         final_states = []
         for global_state in self.strategy:
-            if (
-                self.create_timeout
-                and create
-                and self.time + timedelta(seconds=self.create_timeout) <= datetime.now()
-            ):
-                return final_states + [global_state] if track_gas else None
+            if self.execution_timeout and not create:
```

Comment:
Is this still needed?

Reply:
Yes and no. This code doesn't do much now. But there is a problem when you want to have multiple transactions, as now the execution timeout is used for the entire thing. In another pr I adapt this so it uses a timeout of execution_timeout/#transactions as a limit per transaction